.teamcity/src/subprojects/build/core/CodeStyleVerifyBuild.kt

```diff
@@ -0,0 +1,57 @@
+package subprojects.build.core
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.buildSteps.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.failureConditions.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.triggers.*
+import subprojects.*
+import subprojects.build.*
+
+object CodeStyleVerify : BuildType({
+    id("KtorCodeStyleVerifyKtLint")
+    name = "CodeStyle verify"
+
+    artifactRules = "+:**/build/reports/ktlint/*.txt => plain.tgz"
+
+    vcs {
+        root(VCSCore)
+    }
+
+    steps {
+        gradle {
+            name = "Run ktlint"
+            tasks = "lintKotlin"
+            gradleParams = "-PenableCodeStyle=true"
+            jdkHome = "%env.${jdkVersions.last().env}%"
```

Comment:
A reason for being this and not explicitly a version?

Reply:
Makes no big difference. Can apply `java11` to be sure.